src/auditwheel/wheel_abi.py

```diff
@@ -51,6 +51,13 @@ class NonPlatformWheel(WheelAbiError):
     """No ELF binaries in the wheel"""
 
 
+NOT_PLATFORM_WHEEL_MESSAGE = (
```

Comment:
Can you make this a variable of the class `NonPlatformWheel` please ?

Reply:
> I've taken the liberty of creating a constant with the message, since it was used in 3 different places, but I'm not sure if where I placed it is the correct place. I can remove it if you prefer not having it

I almost suggested it to be done but didn't in the end. Since you've done it, I suggested another place which will also reduce the number of imports.

Reply:
that was by second alternative. Will update